lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -167,7 +167,7 @@ def test_get_bad_keys(self, query_as):
             query_as: Query helper fixture
         """
         response = query_as(
-            {"metadata": ["xyzzy", "plugh", "dataset.owner", "dataset.access"]},
+            {"metadata": "xyzzy,plugh,dataset.owner,dataset.access"},
```

Comment:
You've removed the explicit `list` and replaced it with the stringized list...is there any testing of an explicit `list`, now?

Reply:
The `test_dataset_list` uses a list, albeit with just one element, which doesn't really matter. The point was to make sure at least one test used the string, and I preferred the case that used multiple elements. Nor did it seem necessary to add another test case. The `test_schema.py` cases cover all the variations pretty thoroughly.